spec/aws_support.rb

```diff
@@ -200,11 +200,11 @@ def self.included(context)
     Chef::Provisioning::AWSDriver::Resources.constants.each do |resource_class|
       resource_class = Chef::Provisioning::AWSDriver::Resources.const_get(resource_class)
       resource_name = resource_class.resource_name
-      define_method("update_an_#{resource_name}") do |name, expected_updates={}|
-        AWSSupport::Matchers::UpdateAnAWSObject.new(self, resource_class, name, expected_updates)
+      define_method("update_an_#{resource_name}") do |name, expected_updates={}, &block|
```

Comment:
This is one of my spec changes - you can now pass a block to the `update`, `create`, and `match` matchers which will be provided the `aws_object`.  This allows for custom matchers on the aws object.  See https://github.com/chef/chef-provisioning-aws/pull/358/files#diff-fcb8f9a6302c2b7fa309abf8b8942159R92